node/sc/bridge_manager.go

```diff
@@ -587,44 +587,44 @@ func (bm *BridgeManager) RestoreBridges() error {
 	bm.stopAllRecoveries()
 
 	for _, journal := range bm.journal.cache {
-		var cBridgeInfo *BridgeInfo
-		var pBridgeInfo *BridgeInfo
-		var ok bool
-
 		cBridgeAddr := journal.LocalAddress
 		pBridgeAddr := journal.RemoteAddress
 		bam := bm.subBridge.bridgeAccountManager
 
 		// Set bridge info
-		cBridgeInfo, ok = bm.GetBridgeInfo(cBridgeAddr)
-		if !ok {
+		cBridgeInfo, cok := bm.GetBridgeInfo(cBridgeAddr)
+		pBridgeInfo, pok := bm.GetBridgeInfo(pBridgeAddr)
+		if !cok || !pok {
```

Comment:
As we discussed, we don't need to check this condition to `bridgecontract.NewBridge`.
Let's remove this.

Reply:
Fixed.